client/index.html

```diff
@@ -7,6 +7,14 @@
 </head>
 
 <body>
+  
+
+  <ui-view></ui-view>
+
```

Comment:
Ok, that works, but one thing you should probably fix is move the script tags into the body at the end. It's supposed to be best practice.